### ISSUE TITLE
Upgrade conan node to t4?

### ISSUE THREAD

**Issue by partouf** [compiler-explorer/infra#939]
Is currently a [t2 micro](https://us-east-1.console.aws.amazon.com/ec2/home?region=us-east-1#InstanceTypeDetails:instanceType=t2.micro) instance, but that has "low to moderate" network performance.

It should probably run on a [t4g.micro](https://us-east-1.console.aws.amazon.com/ec2/home?region=us-east-1#InstanceTypeDetails:instanceType=t4g.micro) which has the same vcpu's and memory, but more network perf.. and is cheaper?

Will require, in order:
- [x] Shutdown of current node (so needs to be at a low usage moment and tell ppl libraries might not work)
- [x] Disconnect [vol-0a99526fcf7bcfc11](https://us-east-1.console.aws.amazon.com/ec2/home?region=us-east-1#Volumes:volumeId=vol-0a99526fcf7bcfc11;sort=desc:createTime) from the instance
- [x] Apply terraform with new instance type
- [x] See if it still works... I might have changed some things manually?

**Comment by partouf** [comment]
Should probably also do the same to the SMB instance, but thats easier

**Comment by partouf** [comment]
Oh... t4g is arm based

**Comment by partouf** [comment]
changed smbserver to t3.micro, but will need to think about the conan node...

**Comment by partouf** [comment]
uh oh.. some python upgrade thing running at startup

**Comment by partouf** [comment]
encountered a bunch of issues:
* unattended-upgrade had a lot of things to do
* infra was still on "master", needed switch and pull
* I upgrade node from v12 to v16, but then:
* conanproxy was a mess, so I recloned and did npm install, but then it couldn't download sqlite3 packages anymore. I had to change package.json to use the latest sqlite3 and sqlite packages.
* then I had to change the ownership back to root
* some error had snuck into the startup script https://github.com/compiler-explorer/infra/commit/abf78a081f32b8f08b25cc6659030456c029afbf @mattgodbolt you changed that in https://github.com/compiler-explorer/infra/commit/94a9ad4767667479a55ded3318dd6f6d7ccfe14a#diff-1baf5492458d00b29e201f3678f8ab48408f410e4c30851dd2e7e6627389f45e - is that a new bash thing?
* then I could do `service ce-conan start` without issues

Anyway, hoping to not having to touch it for a couple years to come.
